stable/k8s-spot-rescheduler/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "v0.1.1"
+appVersion: "v0.1.2"
```

Comment:
There is new version available, so we could probably use the latest one https://github.com/pusher/k8s-spot-rescheduler/releases/tag/v0.2.0

Reply:
updated to v0.2.0 with new clusterrole :)